qa/rpc-tests/test_framework/zip244.py

```diff
@@ -288,7 +288,7 @@ def outputs_sig_digest(tx, nHashType, txin):
 def txin_sig_digest(tx, txin):
     digest = blake2b(digest_size=32, person=b'Zcash___TxInHash')
     digest.update(bytes(tx.vin[txin.nIn].prevout))
-    digest.update(bytes(txin.scriptCode))
+    digest.update(ser_string(txin.scriptCode))
```

Comment:
```suggestion
    // TODO: swap the next two fields when ZIP 244 changes become active
    // (see <https://github.com/zcash/zips/pull/577>).
    digest.update(ser_string(txin.scriptCode))
```

Reply:
We already have an issue for this: #5446. We shouldn't make changes to the PR.